pypeit/spectrographs/keck_deimos.py

```diff
@@ -488,7 +488,7 @@ def get_rawimage(self, raw_file, det):
 
         # Return
         exptime = hdu[self.meta['exptime']['ext']].header[self.meta['exptime']['card']]
```

Comment:
Why is this different than gemini_gmos, where you passed back extension 0 and 1.